Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ocaml/opam
+
+WORKDIR learn-ocaml
+
+ADD . .
```

Comment:
It is really a minor detail, but for this line both `ADD . .` and `COPY . .` would do the job, and [this chapter of the doc](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) advises to use `COPY` in this case.

Reply:
Also note that doing `COPY . .` will not only copy in the image all source files of the learn-ocaml repo, but also the .git folder. Is it wanted? If you want to avoid this and thereby reduce a bit the size of the image, you could define a `.dockerignore` with a content similar to `.gitignore`, but also with a line `.git`...

Reply:
(I had a `.dockerignore` locally that I forgot to commit, thanks!)

Actually, you remark on `ADD` and `COPY` was pretty useful: the explanation given in the documentation allowed a better caching of the intermediate steps of building the image. In particular, updating the opam repository and installing the dependencies where taking most of the time of the building process (which was quite annoying during the debug). Thank you!

I'll commit it with other modifications soon.